### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by ValentinaCavinato**

### CODE REVIEW

**Comment by ValentinaCavinato** [comment]
@AndreaCensi

File: book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md [hunk 1]
```diff
@@ -37,13 +31,13 @@ in case it is not, then set it
 
 Now, run the docker container `kmeans` on your duckiebot
 
-    laptop $ docker -H ![ROBOT_NAME].local run -it --memory="800m" --memory-swap="1.8g" --net host --privileged -v /data:/data --name kmeans manishpra/devel-kmeans-unsup:master18 /bin/bash
+    laptop $ docker -H ![ROBOT_NAME].local run -it --memory="800m" --memory-swap="1.8g" --net host --privileged -v /data:/data --name kmeans duckietown/devel-kmeans-unsup:master18 /bin/bash
 
 **Step 2**: Launch the camera_kmeans launch file
 
     duckiebot $ roslaunch pi_camera camera_kmean.launch veh:=![ROBOT_NAME] k:=![k] reduction:=![reduction]
```
Comment (liampaull) [inline book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md]: duckiebot-container ?

File: book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md [hunk 2]
```diff
@@ -54,14 +48,14 @@ camera_kmean.launch file is located under /catkin_ws/src/05-teleop/pi_camera/lau
     laptop $ dts start_gui_tools ![ROBOT_NAME]
     root@user $ rqt_image_view
```
Comment (liampaull) [inline book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md]: laptop-container ?

File: book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md (hunk 1 above)
Comment (manish-pra) [reply book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md]: It is duckietown container. That's correct.

File: book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md (hunk 2 above)
Comment (manish-pra) [reply book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md]: We have to type `dts start_gui_tools ![ROBOT_NAME]` on the laptop it takes us to the root of the laptop and there we open rqt window using rqt_image_view

File: book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md (hunk 1 above)
Comment (liampaull) [reply book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md]: sorry  I meant you can write "duckietbot-container $" in place of "duckiebot $"

File: book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md (hunk 2 above)
Comment (liampaull) [reply book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md]: Same here you can write "laptop-container $"

**Pull request merged**